camayoc/tests/qcs/test_profiles.py

```diff
@@ -113,31 +168,71 @@ def test_negative_create_missing_data(cleanup):
             b) hosts
             c) credential id's
     :expectedresults: Error is thrown and no new profile is created.
-    :caseautomation: notautomated
+    :caseautomation: automated
     """
-    pass
+    cred = HostCredential(client=shared_client, password=uuid4())
+    cred.create()
+    cleanup.append(cred)
+    profile = NetworkProfile(
+        client=api.Client(response_handler=api.echo_handler),
+        hosts=['localhost'],
+        credential_ids=[cred._id],
+    )
 
+    # remove hosts from fields
```

Comment:
I think this comment is wrong, you are removing fields other than just hosts. If you rename data to field then this comment won't be necessary and can be dropped.